### ISSUE TITLE
DTB not found

### ISSUE THREAD

**Issue by ghost** [TeamWin/android_device_oneplus_onyx#11]
With TWRP my device can't access to recovery when i type  fastboot boot recovery.img after a flash, i don't know if the problem come from my fastboot. Somebody for help me, or fix this issue.
`C:\Android>fastboot oem device-info
...
(bootloader)    Device tampered: false
(bootloader)    Device unlocked: true
(bootloader)    Charger screen enabled: true
(bootloader)    Display panel:
(bootloader)    console_enabled: 0
(bootloader)    exec_console_unconsole: 0
OKAY [  0.017s]
finished. total time: 0.019s

C:\Android>fastboot flash recovery recovery.img
target reported max download size of 1073741824 bytes
sending 'recovery' (15920 KB)...
OKAY [  0.507s]
writing 'recovery'...
OKAY [  0.303s]
finished. total time: 0.817s

C:\Android>fastboot boot recovery.img
downloading 'boot.img'...
OKAY [  0.516s]
booting...
FAILED (remote: dtb not found)
finished. total time: `0.556s``

**Comment by carmenfdezb** [comment]
I think that it happens because you're using a version of TWRP no compatible.
What version of OOS are you using? Did you try with others versions of TWRP?

**Comment by ghost** [comment]
I use OOS 3.1 the version of OOS do a change something to the recovery ?

**Comment by carmenfdezb** [comment]
I think you must install TWRP 3.0.3 from https://dl.twrp.me/onyx/
For more information: https://forum.xda-developers.com/oneplus-x/development/unofficial-twrp-3-0-2-0-recovery-onyx-t3471584

**Comment by Pierre-Green** [comment]
Hello I have the same issue with TWRP 3.0.3, did you found a solution ?

**Comment by ghost** [comment]
Yeah it's work with 2.8.7 TWRP but no with > 3.0 so what can i do ? How to update bootloader ?

**Comment by Pierre-Green** [comment]
I don't know it was a question for you because I can't find the solution

**Comment by Keskebeu** [comment]
it does not work with with 2.8.7 either. I was reading somewhere you have to update the bootloader and baseband and it works. I am up to date with my baseband and bootloader so i am assuming that solution is bogus too.
